src/System.Net.Primitives/src/System.Net.Primitives.csproj

```diff
@@ -17,6 +17,7 @@
     <!-- SYSTEM_NET_PRIMITIVES_DLL is required to allow source-level code sharing for types defined within the
          System.Net.Internals namespace. -->
     <DefineConstants>$(DefineConstants);SYSTEM_NET_PRIMITIVES_DLL</DefineConstants>
+    <DefineConstants Condition="'$(TargetGroup)' == 'uapaot'">$(DefineConstants);uapaot</DefineConstants>
```

Comment:
It isn't clear this is going to be useful because we don't build any binaries with 'uapaot' target.  There is no separate 'uapaot' binary in the package that contains System.Net.Primitives.dll.

cc: @weshaggard